doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -141,7 +141,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We have defined a dictionary `sample_kwargs` that will be passed to the `SamplingWrapper` in order to make sure that all refits use the same sampler parameters. We follow the same pattern with `az.from_numpyro`."
+    "We have defined a dictionary `sample_kwargs` that will be passed to the `SamplingWrapper` in order to make sure that all\n",
+    "refits use the same sampler parameters. We follow the same pattern with {func}`~az.from_numpyro <arviz.from_numpyro>`."
```

Comment:
Remove the `~` from {func}`~az.from_numpyro <arviz.from_numpyro>`

Reply:
that space is still there?